setup.sh

```diff
@@ -35,7 +27,13 @@ clean(){
 install(){
   echo 'Installing project dependencies...'
   npm install
-  bower install --config.interactive=false
+  npm install --save --save-exact box-sizing-polyfill@0.1.0 jquery.easing@1.3.2 normalize-css@2.3.1 jquery@1.11.3 normalize-legacy-addon@0.1.0
```

Comment:
Per @amymok's [comment below](https://github.com/cfpb/college-costs/pull/155#discussion_r63422397), we do in fact need this here until Capital Framework deals with shrinkwrapping dependencies better. [It seems that's gonna be dealt with soon, though.](https://github.com/cfpb/capital-framework/issues/312)